roles/kubernetes/master/templates/apiserver-webhook-config.yaml.j2

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Config
+clusters:
+- cluster:
+ server: {{ audit_webhook_server_url }}
+name: logstash
```

Comment:
I guess "logstash" should not be hardcoded or possible have a more generic name like "auditsink"?

Reply:
Good idea, fix it